canister/candid.did

```diff
@@ -1,6 +1,7 @@
 type network = variant {
   mainnet;
   testnet;
+  testnet4;
```

Comment:
I'm not sure adding this field is the right approach.

What if the Bitcoin testnet is (finally) upgraded to version 4? It would then be weird to have a `testnet` variant that is unused.
Of course, the downside is that people may be confused if we simply call the new version `testnet`.
I'm leaning towards option #2 anyway because the current testnet is useless for many people and so I'm hoping that there will be an upgrade.
What do you think?

Reply:
So you're proposing to keep only 2 variants `mainnet` and `testnet` but to use `testnet4` under the hood?
That's an interesting idea, and I agree it'd would reduce the confusion (for not having several testnets).

Let me try to apply this idea into code.

Reply:
It can also cause confusion: If a developer makes a test transaction and then tries to find it on a Bitcoin testnet (not testnet4) explorer, he/she won't find it.

This risk could be mitigated by documenting everywhere very clearly that `testnet4` is used.

Reply:
I have updated the code to re-use `testnet` variant and using `testnet4` under the hood. PTAL